src/aap_eda/services/project/imports.py

```diff
@@ -61,13 +61,12 @@ def wrapper(self: ProjectImportService, project: models.Project):
         try:
             func(self, project)
             project.import_state = models.Project.ImportState.COMPLETED
-        except Exception as e:
+        except ScmEmptyError as e:
             # if a project is empty, sync status should show completed
-            project.import_state = (
-                models.Project.ImportState.COMPLETED
-                if "Project folder is empty" in e.args[0]
-                else models.Project.ImportState.FAILED
-            )
+            project.import_state = models.Project.ImportState.COMPLETED
+            project.import_error = str(e)
```

Comment:
It doesn't make sense to set an "import error" if the state is not error. Here I see a potential inconsistency.

Reply:
As discussed in the SU. This field should be renamed in another issue when it is time to change UI together.